lib/plugins/aws/deploy/lib/listStacks.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+const findAndGroupStacks = require('../utils/findAndGroupStacks');
+
+module.exports = {
+  listStacks() {
+    const service = this.serverless.service.service;
+    const stage = this.options.stage;
+
+    return this.sdk.request('S3',
```

Comment:
Could you replace this with a usage of the recently introduced `AwsProvider` plugin usage (Then it should be something like `this.provider.request`).

Here's how @eahefnawy has done it in the custom resource merging PR fix:
https://github.com/serverless/serverless/pull/2385/commits/f0804d244f2fbb353a944468225d17f663c5d916